test/smoke/src/areas/positron/editor/fast-execution.test.ts

```diff
@@ -35,7 +33,27 @@ export function setup(logger: Logger) {
 
 				await app.workbench.quickaccess.openFile(join(app.workspacePathOrFolder, 'workspaces', 'fast-statement-execution', FILENAME));
 
-				for (let i = 1; i < 12; i++) {
+				let previousTop = -1;
+
+				// Note that this outer loop iterates 10 times.  This is because the length of the
+				// file fast-execution.r is 10 lines.  We want to be sure to send a Control+Enter
+				// for every line of the file
+				for (let i = 1; i < 11; i++) {
```

Comment:
```suggestion
				// file fast-execution.r is 10 lines.  We want to be sure to send a Control+Enter
				// for every line of the file
				for (let i = 0; i < 10; i++) {
```
Since the iterator variable isn't being used it's more typical to start loop at 0, at least it makes it easier to see the fact it loops 10 times.